packages/components/multiselect/README.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Multiselect'
+slug: /components/multiselect/
+github: 'https://github.com/contentful/forma-36/tree/main/packages/components/multiselect'
+storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-multiselect--basic'
+typescript: ./src/Multiselect.tsx
+---
+
+Search through a long list of options.
+
+## Import
+
+```jsx static=true
+import { Multiselect } from '@contentful/f36-components';
+// or
+import { Multiselect } from '@contentful/f36-multiselect';
+```
+
+## Examples
+
+### Basic usage
+
+The Multiselect requires 3 props to work:
+
+- `items`: It’s an array containing the items that will be shown as selectable options when the user types something in the `TextInput`.
+- `onInputValueChange`: This function will be called every time the user types something in the input.
+  The component will pass the `item`, which the filter method is currently iterating over, and the `inputValue` prop of the `TextInput` component.
+- `onSelectItem`: This function is called when the user selects one of the options of the list. The component will pass the selected item as an argument to the function.
+
+An `Multiselect` with a list of spaces will look like this:
+
+```jsx file=./examples/MultiselectBasicUsageExample.tsx
+
+```
+
+### Using objects as `items`
+
+We showed how to create an Multiselect with an array of string but it’s also possible to use other types of data as `items`.
+A very common way of using the Multiselect is with objects and for that, with a few changes to the previous example this can be done:
```

Comment:
```suggestion
We showed how to create a Multiselect with an array of strings but it’s also possible to use other data types as `items`.
A widespread way of using Multiselect is with objects and for that, with a few changes to the previous example this can be done:
```

_Suggestions by Grammarly_